README.md

```diff
@@ -61,7 +61,7 @@ Give a ⭐️ if you like this project!
 ## Acknowledgments
 
 - Thanks to Microverse for giving this opportunity to learn ...
-- Hat tip : Mr.Aamir Khan and my all my colegues for this week.
+- Hat tip : Mr.Aamir Khan and all my Microverse colleague for this week.
 
```

Comment:
- [ ] [OPTIONAL] Your README looks great but it would be much better if you add the Screenshot of the landing page of your website 😉
- [ ] [OPTIONAL] Your project is updated when **I run it locally but not the live version**. Therefore, I would highly recommend you **update the live version of your website** 😉

Reply:
yes, I was about to do so, but due to the deadline time, I skipped that step.
But in future definitely, I would like to improve.
And thanks for the suggestion.